### PR TITLE
Added steps/nnet3/train_dnn.py and made changes to swbd/tdnn recipe t…

### PR DESCRIPTION
**Pull request by vijayaditya**
…o use this new python training script

Running an experiment to ensure that I can reproduce the results.

### CODE REVIEW

**Comment by vijayaditya** [comment]
Tested all code blocks. I don't want to waste GPUs in trying to recreate results. However I have checked the logs and all parts of the code seem to be running fine.

I will not be making any more changes. This is ready for merge after review.

File: egs/swbd/s5c/local/nnet3/run_tdnn.sh [hunk 1]
```diff
@@ -41,26 +44,52 @@ ali_dir=exp/tri4_ali_nodup$suffix
...
-    --splice-indexes "-2,-1,0,1,2 -1,2 -3,3 -7,2 0" \
-    --feat-type raw \
-    --online-ivector-dir exp/nnet3/ivectors_${train_set} \
-    --cmvn-opts "--norm-means=false --norm-vars=false" \
-    --initial-effective-lrate 0.0017 --final-effective-lrate 0.00017 \
-    --cmd "$decode_cmd" \
-    --relu-dim 1024 \
-    data/${train_set}_hires data/lang $ali_dir $dir  || exit 1;
+  steps/nnet3/train_dnn.py --stage=$train_stage \
+    --command="$decode_cmd" \
```
Comment (danpovey) [inline egs/swbd/s5c/local/nnet3/run_tdnn.sh]: I think it's better to use "cmd" as it would then be the same as the shell scripts.

Comment (vijayaditya) [inline egs/swbd/s5c/local/nnet3/run_tdnn.sh]: done.

**Comment by danpovey** [comment]
Thanks!  Merging.

**Pull request merged**